src/test/java/com/epam/MobileCategoriesList.java

```diff
@@ -0,0 +1,29 @@
+package com.epam;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class MobileCategoriesList {
+
+    @Test
+    @DisplayName("Verify Categories list is active(expanded) in when web page is in mobile size")
+    public void mobileCategoriesList() {
+        System.setProperty("webdriver.chrome.driver", "chromedriver.exe");
+        WebDriver driver = new ChromeDriver();
+        driver.get("http://automationpractice.com/index.php");
+
+        Dimension d = new Dimension(480, 620);
+        driver.manage().window().setSize(d);
+        driver.findElement(By.xpath("//header/div/div[@class='container']/div[@class='row']/div[@id='block_top_menu']/div[@class='cat-title']")).click();
+
+        //to verify is the list is active we are checking style attribute. If it has value block == active if none == non active
+        driver.findElement(By.xpath("//header/div/div[@class='container']/div[@class='row']/div[@id='block_top_menu']/ul[contains(@style, 'display: block;')]"));
```

Comment:
same as above, it's overcomplicated without tan obvious reason. the id is a great help, and the ul's style parameter seems not to add anything. did I miss the point in adding the rest?

Reply:
agree, but unfortunately due to the site is not longer available I can not verify the fix so will leave it as it is so far.